src/redux/innsynsdata/innsynsdataReducer.ts

```diff
@@ -98,10 +98,10 @@ export enum InnsynsdataActionTypeKeys {
     SETT_STATUS_FOR_ETTERSENDELSESFIL = "innsynsdata/SETT_STATUS_FOR_ETTERSENDELSESFIL",
     OPPDATER_SAKSDETALJER = "innsynsdata/OPPDATER_SAKSDETALJER",
     SETT_REST_STATUS_SAKSDETALJER = "innsynsdata/SETT_REST_STATUS_SAKSDETALJER",
-    OPPGAVE_VEDLEGSOPPLASTING_FEILET = "innsynsdata/OPPGAVE_VEDLEGSOPPLASTING_FEILET",
-    OPPGAVE_OPPLASTING_FEILET = "innsynsdata/OPPGAVE_OPPLASTING_FEILET",
-    OPPGAVE_OPPLASTING_BACKEND_FEILET = "innsynsdata/OPPGAVE_OPPLASTING_BACKEND_FEILET",
-    OPPGAVE_OPPLASTING_BACKEND_FEILET_PGA_VIRUS = "innsynsdata/OPPGAVE_OPPLASTING_BACKEND_FEILET_PGA_VIRUS",
+    FILE_ATTACHMENTSUPLOADING_FAILED = "innsynsdata/FILE_VEDLEGSOPPLASTING_FEILET",
```

Comment:
Hvis vi skal bruke engelsk navn på key her, så hadde det vært flott om vi brukte samme på nøkkelen også. :) Også her stemmer jeg for å bruke `*UPLOAD` i stedet for `*UPLOADING`